.gitignore

```diff
@@ -1,4 +1,4 @@
 keys/id_rsa
 keys/id_rsa.pub
 keys.tgz
-
+account_file.json
```

Comment:
This is needed because when running locally you need to have the account file in the PWD, right?